src/Changelogs/ModuleLearningObjectGateway.ts

```diff
@@ -0,0 +1,22 @@
+import { LearningObjectGateway } from './LearningObjectGateway';
+import { LearningObjectAdapter } from '../LearningObjects/adapters/LearningObjectAdapter';
+import { LearningObjectSummary } from '../shared/types';
+import { Requester } from './typings';
+
+export class ModuleLearningObjectGateway extends LearningObjectGateway {
```

Comment:
Use `implements` not `extends`. `ModuleLearningObjectGateway` is not a sub-class.
```suggestion
export class ModuleLearningObjectGateway implements LearningObjectGateway {
```